routes/index.js

```diff
@@ -0,0 +1,14 @@
+var express = require("express");
+var router = express.Router();
+
+/* GET home page. */
+router.get("/", (req, res) => {
+  const name = req.query.name;
+  if (name === undefined) {
+    res.status(400).json({ name: "null", message: "You should write name" });
+  } else {
+    res.render("index", { name });
```

Comment:
What we will render in case of empty name (empty string in param)?

Reply:
we will see mistake because page write only when we do name